### ISSUE TITLE
DB Build issue with msbuild 17.5

### ISSUE THREAD

**Issue by DavidWiseman** [trimble-oss/dba-dash#554]
DB build fails with msbuild 17.5.  Issue logged [here](https://github.com/microsoft/DacFx/issues/236)

Might need to consider workarounds

**Comment by DavidWiseman** [comment]
Fix to specify version 17.4 no longer seems to be working.

```yml
     - name: Add msbuild to PATH
        uses: microsoft/setup-msbuild@v1.3
        with:
          vs-version: '[17.4,17.5)'
```

**Comment by DavidWiseman** [comment]
Security Scan action still seems to be working.  It's running with an older version of msbuild:

Microsoft (R) Build Engine version 16.11.2+f32259642 for .NET Framework

This workflow uses:
`runs-on: windows-2019`
instead of 
`runs-on: windows-2019`

This might not be a reliable fix but worth noting.

**Comment by DavidWiseman** [comment]
Included in 2.36.1 🎉
